cg/cli/upload.py

```diff
@@ -78,6 +88,8 @@ def upload(context, family_id):
             click.echo(click.style(f"{family_id}: analysis uploaded!", fg='green'))
 
 
+upload.add_command(trending_command)
```

Comment:
Please add this to the end of the file instead. No one is expecting an add_command between function definitions ;)

![image](https://user-images.githubusercontent.com/239608/65821799-c99cfc80-e23a-11e9-893a-ad9655d0d16d.png)

Reply:
:) okej